### PR TITLE
Lesson03 make: Eugene.Ovdiyuk 

### PR DESCRIPTION
**Pull request by ovdiyuk**
Makefile home task:

Makefile is finished for 3 folders files as required
Corrected make_simplest project
See https://github.com/Kernel-GL-HRK/gl-kernel-training-2018/pull/38 that you commented.

### CODE REVIEW

**Comment by ovdiyuk** [comment]
@yekovalyov you previously reviewed https://github.com/Kernel-GL-HRK/gl-kernel-training-2018/pull/38
I made all correction. I won't take much time to take a look it again to approve. Please.

File: 03-make/make_homework/Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+TXT:=cat #computes variable value once and just stores it
+TARGET?=sup.txt #if the variable was not defined
+
+all : stol.o
```
Comment (yekovalyov) [inline 03-make/make_homework/Makefile]: Does it work as expected?

File: 03-make/make_homework/Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	cp $< $(TARGET)
```
Comment (yekovalyov) [inline 03-make/make_homework/Makefile]: What is the idea of this command?

File: 03-make/make_homework/Makefile (hunk 1 above)
Comment (ovdiyuk) [reply 03-make/make_homework/Makefile]: Yes, it works as I expected, I checked and added ".log" file to the last commit. If this not as you expected could you please send me the right console output and I'll refactor this.

File: 03-make/make_homework/Makefile (hunk 2 above)
Comment (ovdiyuk) [reply 03-make/make_homework/Makefile]: The comment was added in the code. This just copies stol.o to the $TARGET.
I agree that this extra operation and I can do it directly in the stol.o rule. But I'm learning and wanted to show you I know this command :)

**Pull request merged**